api/app/schemas/hfi_calc.py

```diff
@@ -181,3 +181,26 @@ class HFIResultResponse(BaseModel):
     planning_area_fire_starts: Mapping[int, List[FireStartRange]]
     # Indicate whether the request used to generate this response  was saved to the database.
     request_persist_success: bool
+
+
+class PrepCyclePDFData(BaseModel):
+    """ Data needed for prep cycle PDF sheet """
+    planningAreaName: str
+    # Station dailies grouped by station code containing the dailies for each day in the prep cycle
+    dailies: Mapping[int, List[StationDaily]]
+
+
+class StationPDFData(StationDaily, WeatherStation):
```

Comment:
I didn't even know you could do that! I thought is was illegal. Beware multiple inheritance. Composition rather, maybe?

Reply:
Yeah it's so nice, removes a lot of boilerplate. Not sure if composition is possible.

Reply:
I had a lot of pain in c++ due to multiple inheritance, so I was quite happy that Java didn't allow it.

Reply:
Yes I agree on composition over inheritance. The above allows me to merge a dict representation of a `StationDaily` and a `WeatherStation` and just hand it to the constructor. Composing and adding fields would add more bookkeeping. What do you think? Keep this or no?

Reply:
I mean - it works - keep it.